contrib/epee/src/mlocker.cpp

```diff
@@ -59,7 +65,8 @@ static void do_lock(void *ptr, size_t len)
 {
 #if defined HAVE_MLOCK
   int ret = mlock(ptr, len);
-  if (ret < 0)
+  // any subsequent failure will be silenced
```

Comment:
Sorry, I meant in the log message, so people who see this know there might be more :D

Reply:
That makes more sense actually. Updated the warning message.